web/blob/static/js/status.js

```diff
@@ -0,0 +1,26 @@
+/* globals PATH_PREFIX, jQuery */
+(function ($, undefined) {
+    'use strict';
+
+    $('#reload-conf-btn').on('click', function () {
+        var elBtn;
+        $(this).promise().then(function (btn) {
+            elBtn = btn;
+            elBtn.text('Waiting...').prop('disabled', true);
+            return $.post(PATH_PREFIX + '/-/reload');
+        }).then(function () {
+            elBtn.text('Loaded Successfully!')
```

Comment:
Let's say "Reloaded Successfully"?

Actually, unrelated to those UI changes, I noticed that the HTTP endpoint even returns a `200` if the config reload fails, due to the asynchronicity of the reload (the HTTP endpoint just sends a reload request into the channel, but has no way of knowing whether it actually succeeded).

So all we strictly know here is that the reload was *triggered*. So how about just "Triggered Reload" (I guess "Successfully Triggered Reload" would become quite long)? And in the failure case, "Triggering Reload Failed"?